test/util/testnode/config.go

```diff
@@ -153,7 +153,7 @@ func DefaultTendermintConfig() *tmconfig.Config {
 	// Override the mempool's MaxTxBytes to allow the testnode to accept a
 	// transaction that fills the entire square. Any blob transaction larger
 	// than the square size will still fail no matter what.
-	maxTxBytes := appconsts.DefaultSquareSizeUpperBound * appconsts.DefaultSquareSizeUpperBound * appconsts.ContinuationSparseShareContentSize
+	maxTxBytes := appconsts.DefaultUpperBoundMaxBytes
```

Comment:
👍  thanks for the refactor to use a defined constant for this value.

[no change needed][separate issue]: why does the default here not match the default used on mainnet? IMO only one test (max square size test) should override this value and the rest of our tests should use the `appconsts.MaxTxBytes`

Reply:
Yeah I agree that there should be a single config that we use in all our tests (and only change them when they need to be changed). I think this should be addresses as a follow up

Reply:
+1 as a follow-up